operator/config/default/kustomization.yaml

```diff
@@ -38,12 +38,35 @@ components:
   #- ../prometheus
   # [GRAFANA] To generate configmap for provision grafana dashboard
   - ../grafana
+  # [Load Test] To generate default prometheus operator related resources for load testing
+#  - ../load_test
+
+patchesJson6902:
+  - target:
+      version: v1
+      kind: ConfigMap
+      name: manager-dashboard
+    patch: |-
+      - op: replace
+        path: /metadata/namespace
+        value: kyma-system
+  - target:
+      version: v1
+      kind: ConfigMap
+      name: controller-runtime-metrics-dashboard
+    patch: |-
+      - op: replace
+        path: /metadata/namespace
+        value: kyma-system
```

Comment:
I have to write this patches specifically to overwrite the namespace for those dashboard configmap